tox.ini

```diff
@@ -39,11 +38,11 @@ deps=
 
     cov: -r{toxinidir}/requirements/test-django.txt
 
-	django32: Django ~= 3.2
 	django41: Django ~= 4.1
 	django42: Django ~= 4.2
 	django50: Django ~= 5.0
-	django51: Django ~= 5.1rc1
+	django51: Django ~= 5.1
+	django52: Django ~= 5.2rc1
```

Comment:
Would this fail if we had "5.2" instead of "5.2rc1"?

Reply:
Yes.